pkg/controller/certificates/certificate_controller.go

```diff
@@ -142,7 +145,7 @@ func (cc *CertificateController) processNextWorkItem(ctx context.Context) bool {
 		if _, ignorable := err.(ignorableError); !ignorable {
 			utilruntime.HandleError(fmt.Errorf("Sync %v failed with : %v", cKey, err))
 		} else {
-			klog.V(4).Infof("Sync %v failed with : %v", cKey, err)
+			klog.FromContext(ctx).V(4).Info("Sync certificate request failed", "csr", cKey, "err", err)
```

Comment:
Use `.Error`?

Reply:
That would be a much higher log level and imply that admins should be notified when this line is reached. That wasn't the case before.

Reply:
I mean `.V(4).Error(`

Reply:
This log level has always been `.V(4).Info`, I'm not sure if this needs to be changed, I prefer to keep it that way for this PR.

Reply:
`V(4).Error` is the same as `Error`: `Error` calls have no verbosity level.